coverage/lcov.info

```diff
@@ -0,0 +1,27 @@
+TN:
+SF:/Users/chisomonwuchekwa/Documents/simsFront/freyja-ah-frontend/src/components/header.js
+FNF:0
+FNH:0
+LF:0
+LH:0
+BRF:0
+BRH:0
+end_of_record
+TN:
+SF:/Users/chisomonwuchekwa/Documents/simsFront/freyja-ah-frontend/src/pages/signup.js
+FNF:0
+FNH:0
+LF:0
+LH:0
+BRF:0
+BRH:0
+end_of_record
+TN:
+SF:/Users/chisomonwuchekwa/Documents/simsFront/freyja-ah-frontend/src/routes/routes.js
+FNF:0
+FNH:0
+LF:0
+LH:0
+BRF:0
+BRH:0
+end_of_record
```

Comment:
I see you already added the `coverage` directory to `.gitignore` but you still have to remove this ones that you already committed